tools/export/vscode/vscode.md

```diff
@@ -0,0 +1,40 @@
+# VSCode exporter
```

Comment:
Lets get rid of this and have a proper docs page in the handbook under exporters